addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -317,6 +323,49 @@ def __gather_clearcoat_extension(blender_material, export_settings):
 
     return Extension('KHR_materials_clearcoat', clearcoat_extension, False)
 
+# See: https://github.com/sobotka/blender/blob/4dd1068a5789097b50eab159adc995906fa5ea84/source/blender/gpu/shaders/material/gpu_shader_material_principled.glsl#L2-L6
```

Comment:
Better to link to official repository here:
https://developer.blender.org/diffusion/B/browse/master/source/blender/gpu/shaders/material/